### PR TITLE
Update to quest 3.7 and including option to build for cuda in build.rs

### PR DESCRIPTION
**Pull request by nfwvogt**

### CODE REVIEW

File: quest-sys/Cargo.toml [hunk 1]
```diff
@@ -26,10 +26,12 @@ openmp-sys ={version = "1.2.0", features=["static"], optional=true}
 
 [build-dependencies]
 # cmake = {version ="0.1", features=[]}
-bindgen = {version="0.64", optional=true}
+bindgen = {version="0.69", optional=true}
```
Comment (mlodi-hqs) [inline quest-sys/Cargo.toml:29]: bindgen says that the minimum Rust supported version is 1.60 (https://github.com/rust-lang/rust-bindgen#msrv), but tests are green, so...

File: quest-sys/build.rs [hunk 2]
```diff
@@ -68,45 +68,101 @@ fn build_with_cc(out_dir: PathBuf) -> PathBuf {
...
+            .include(custate_include_lib)
+            .files(files)
+            .define("MULTITHREADED", "0")
+            .define("USE_CUQUANTUM", "1")
+            .define("USE_HID", "0")
+            .opt_level(2)
+            .debug(false)
+            .warnings(false)
+            .out_dir(out_path.clone())
+            .cuda(true)
```
Comment (mlodi-hqs) [inline quest-sys/build.rs:163]: Must this be true in here as well?

File: quest-sys/Cargo.toml (hunk 1 above)
Comment (nfwvogt) [reply quest-sys/Cargo.toml:29]: Good catch but in this case we will up the MSRV.

File: quest-sys/build.rs (hunk 2 above)
Comment (nfwvogt) [reply quest-sys/build.rs:163]: Yes that also uses the nvcc compiler.

**Pull request merged**